client_app/src/components/admin/AdminDashboard.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import AddShelterForm from './AddShelterForm';
+import ShelterList from './ShelterList';
```

Comment:
I think you need to fix the import path here too.
<img width="1690" alt="Screenshot 2025-03-04 at 11 34 27 AM" src="https://github.com/user-attachments/assets/9d1c8733-91a5-4e45-9a24-7afffe08a452" />